components/AddRecommendations.js

```diff
@@ -1,9 +1,8 @@
-import React, { Fragment, useEffect, useState } from 'react';
+import React, { Fragment, useState } from 'react';
```

Comment:
See fragment comment below


```suggestion
import React, { useState } from 'react';
```